<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDelimitedTextSourceSelectBase</class>
 <widget class="QDialog" name="QgsDelimitedTextSourceSelectBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Layer from a Delimited Text File</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabelFileName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>File Name</string>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtFilePath">
        <property name="toolTip">
         <string>Full path to the delimited text file</string>
        </property>
        <property name="whatsThis">
         <string>Full path to the delimited text file. In order to properly parse the fields in the file, the delimiter must be defined prior to entering the file name. Use the Browse button to the right of this field to choose the input file.</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabelLayerName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Layer name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtLayerName">
        <property name="toolTip">
         <string>Name to display in the map legend</string>
        </property>
        <property name="whatsThis">
         <string>Name displayed in the map legend</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btnBrowseForFile">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Browse to find the delimited text file to be processed</string>
        </property>
        <property name="whatsThis">
         <string>Use this button to browse to the location of the delimited text file. This button will not be enabled until a delimiter has been entered in the &lt;i&gt;Delimiter&lt;/i&gt; box. Once a file is chosen, the X and Y field drop-down boxes will be populated with the fields from the delimited text file.</string>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_select_delimiters">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="delimiterSelection">
        <property name="text">
         <string>Selected delimiters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="delimiterRegexp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The delimiter is a regular expression</string>
        </property>
        <property name="whatsThis">
         <string>The delimiter is a regular expression</string>
        </property>
        <property name="text">
         <string>Regular expression</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="delimiterPlain">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>The delimiter is taken as is</string>
        </property>
        <property name="whatsThis">
         <string>The delimiter is taken as is</string>
        </property>
        <property name="text">
         <string>Plain characters</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="cbxDelimTab">
          <property name="text">
           <string>Tab</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbxDelimSpace">
          <property name="text">
           <string>Space</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="cbxDelimComma">
          <property name="text">
           <string>Comma</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="cbxDelimSemicolon">
          <property name="text">
           <string>Semicolon</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="cbxDelimColon">
          <property name="text">
           <string>Colon</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDelimiter">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>32767</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Delimiter to use when splitting fields in the text file. The delimiter can be more than one character.</string>
        </property>
        <property name="whatsThis">
         <string>Delimiter to use when splitting fields in the delimited text file. The delimiter can be 1 or more characters in length.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,2">
      <item row="1" column="1">
       <widget class="QSpinBox" name="rowCounter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start import at row</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="gridFrame_2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="geomTypeXY">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string>The file contains X and Y coordinate columns</string>
          </property>
          <property name="text">
           <string>X Y fields</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabelx">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;X field&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbXField">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Name of the field containing x values</string>
          </property>
          <property name="whatsThis">
           <string>Name of the field containing x values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabely">
          <property name="text">
           <string>&lt;p align=&quot;right&quot;&gt;Y field&lt;/p&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbYField">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Name of the field containing y values</string>
          </property>
          <property name="whatsThis">
           <string>Name of the field containing y values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="geomTypeWKT">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>The file contains a well known text geometry field</string>
        </property>
        <property name="text">
         <string>WKT field</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbWktField">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Name of the field containing y values</string>
        </property>
        <property name="whatsThis">
         <string>Name of the field containing y values. Choose a field from the list. The list is generated by parsing the header row of the delimited text file.</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>177</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="decimalPoint"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Decimal point</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sample text</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QTableWidget" name="tblSample"/>
   </item>
   <item row="6" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtFilePath</tabstop>
  <tabstop>btnBrowseForFile</tabstop>
  <tabstop>txtLayerName</tabstop>
  <tabstop>delimiterSelection</tabstop>
  <tabstop>cbxDelimTab</tabstop>
  <tabstop>cbxDelimSpace</tabstop>
  <tabstop>cbxDelimComma</tabstop>
  <tabstop>cbxDelimSemicolon</tabstop>
  <tabstop>cbxDelimColon</tabstop>
  <tabstop>delimiterPlain</tabstop>
  <tabstop>delimiterRegexp</tabstop>
  <tabstop>txtDelimiter</tabstop>
  <tabstop>rowCounter</tabstop>
  <tabstop>geomTypeXY</tabstop>
  <tabstop>cmbXField</tabstop>
  <tabstop>cmbYField</tabstop>
  <tabstop>geomTypeWKT</tabstop>
  <tabstop>cmbWktField</tabstop>
  <tabstop>tblSample</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDelimSpace</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDelimTab</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDelimSemicolon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDelimComma</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtDelimiter</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delimiterSelection</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbxDelimColon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
